.env-dist

```diff
@@ -15,7 +15,8 @@ STATE=Georgia
 CITY=Atlanta
 
 #
-# Database configurations
+# Database configurations 
+# /!\ POSTGRES_USER & PG_USER must be the same user or Celery will fail to start
```

Comment:
What are the front and backslash for here actually?

Reply:
To make a warning ⚠️ panel 😉